js_modules/dagster-ui/packages/ui-core/src/search/useIndexedDBCachedQuery.tsx

```diff
@@ -108,7 +108,7 @@ export class IndexedDBQueryCache<TQuery, TVariables extends OperationVariables>
     this.version = version;
     this.variables = variables;
     this.queryFn = queryFn;
-    this.cacheManager = new CacheManager<TQuery>(key);
+    this.cacheManager = getCacheManager(key);
```

Comment:
This will get us a lot more juice.

Reply:
(Previously we were creating a new one every time... which means a fresh pull from indexeddb... now we pull it once and just reuse whats in memory the rest of the time).